nvtabular/ops/base.py

```diff
@@ -45,7 +45,7 @@ class Operator:
     Base class for all operator classes.
```

Comment:
Since the file has been renamed to `base.py`, I wonder if we should also rename this class to `BaseOperator` to match. 🤔

Reply:
I think it's just a style thing? I don't have a strong preference either way.